src/components/layout/Navigation.js

```diff
@@ -62,7 +63,7 @@ export default function Navigation() {
   const [moreDropdownActive, updateMoreDropdownActive] = useState(false);
   const [toolsDropdownActive, updateToolsDropdownActive] = useState(false);
   return (
-    <BorderedNav height={"60px"}>
+    <BorderedNav>
```

Comment:
🤔 Was that height prop even doing anything? I'd expect we would have had to do something like `style={height: 60px}` OR take in the `prop` within the styled component, but we were doing neither of those! Anyway, nice catch!

Reply:
We do -> https://github.com/HandshakeAlliance/HNScan/blob/staging/src/components/styles/NavBar/NavBar.js#L6

this just mimics the behavior the BulmaReact library had, but the default value is 3.25em which is (3.25 * 16) == ~52. So roughly the same height. In this particular scenario its a non-change bc it sets min-height and not height, so with the logo being larger than 52px it stays the same.

Reply:
oh nice eye. I didn't realize BorderedNav was an extension of NavBar. Makes sense 👍

Reply:
Oops, didn't realized this was being used as a prop. Reverting back to the original in my next commit